front/components/vaults/VaultDataSourceViewContentList.tsx

```diff
@@ -157,6 +158,22 @@ export const VaultDataSourceViewContentList = ({
               }}
             />
           </>
+        ) : dataSourceView.category === "managed" ? (
+          <div className="flex flex-col items-center gap-2 text-sm text-element-700">
+            <span>No data sources were added yet.</span>
+            <Button
+              variant="primary"
+              icon={PlusIcon}
+              label="Add data"
+              onClick={() => {
+                void router.push(
+                  `/w/${owner.sId}/data-sources/vaults/${vault.sId}/categories/${dataSourceView.category}`
```

Comment:
So you redirect to the parent page? 
Shouldn't we allow configuring the connection when we're on the root page of the connection instead?

Reply:
Yes if we're at the root of the Connection. You can check with @Duncid before implementing but this would seem legit (and better UX than putting you back on the list of Connections where you have to understand you have to click on "Manage").

Reply:
Seems indeed better, like it 😇